js/shaders/scatter-fragment.glsl

```diff
@@ -33,39 +33,43 @@ vec2 rotate_xy(vec2 x, float angle) {
 
 void main(void) {
     vec2 pixel = (vUv - 0.5) * (pixel_size + 2.0 * stroke_width);
+    float fill_weight = 0.0;
 
 #if FAST_DRAW == FAST_CIRCLE
-    float r = length(pixel);
-    float fill_weight = 1.0 - smoothstep(pixel_size/2.0 - stroke_width - SMOOTH_PIXELS, pixel_size/2.0 - stroke_width + SMOOTH_PIXELS, r);
+    float r = length(pixel); // This uses `sqrt` which is slow, there might be room for performance improvements
```

Comment:
Yes, we could work in 'squared' space.

Reply:
Yes that's what I did later on :) on this commit 516013c